### PR TITLE
Gripper rotation, ROSGeometry addition

### PR DESCRIPTION
**Pull request by at669**
- Rotates the Niryo gripper when picking up the cube
- Uses ROSGeometry instead of manual conversion
- Updated instructions accordingly

### CODE REVIEW

**Review by mpinol — APPROVED** [review]

File: tutorials/pick_and_place/2_ros_tcp.md [hunk 1]
```diff
@@ -104,23 +104,15 @@ To enable communication between Unity and ROS, a TCP endpoint running as a ROS n
...
       sourceDestinationMessage.pick_pose = new RosMessageTypes.Geometry.Pose
       {
-         position = new Point(
-               target.transform.position.z,
-               -target.transform.position.x,
-               target.transform.position.y
-         ),
-         orientation = pickOrientation
+         position = target.transform.position.To<FLU>(),
+         orientation = Quaternion.Euler(90, target.transform.eulerAngles.y, 0).To<FLU>()
```
Comment (mpinol) [inline tutorials/pick_and_place/2_ros_tcp.md:108]: Is there a comment before this line describing the significance of the hardcoded 90 and 0 values?

Comment (at669) [reply tutorials/pick_and_place/2_ros_tcp.md:109]: just added one to be analogous to the comment for `pickOrientation`!

File: tutorials/pick_and_place/Scripts/TrajectoryPlanner.cs [hunk 2]
```diff
@@ -15,10 +20,10 @@ public class TrajectoryPlanner : MonoBehaviour
     // Hardcoded variables 
     private readonly float jointAssignmentWait = 0.1f;
     private readonly float poseAssignmentWait = 0.5f;
-    private readonly float pickPoseOffset = 0.1f;
+    private readonly Vector3 pickPoseOffset = new Vector3(0, 0.1f, 0);
```
Comment (LaurieCheers-unity) [inline tutorials/pick_and_place/Scripts/TrajectoryPlanner.cs:23]: Could make the intent more explicit by saying `pickPoseOffset = Vector3.up * 0.1f;`

File: tutorials/pick_and_place/Scripts/TrajectoryPlanner.cs [hunk 3]
```diff
@@ -15,10 +20,10 @@ public class TrajectoryPlanner : MonoBehaviour
...
     
     // Assures that the gripper is always positioned above the target cube before grasping.
-    private readonly RosQuaternion pickOrientation = new RosQuaternion(0.5,0.5,-0.5,0.5);
+    private readonly Quaternion pickOrientation = new Quaternion(-0.5f,-0.5f,0.5f,-0.5f);
```
Comment (LaurieCheers-unity) [inline tutorials/pick_and_place/Scripts/TrajectoryPlanner.cs:26]: Only a monster writes their quaternions by components :-)
Can this be expressed with Quaternion.Euler?

**Review by LaurieCheers-unity — APPROVED** [review]

**Pull request merged**